pyserini/search/pysearch.py

```diff
@@ -252,3 +252,25 @@ def get_topics(collection_name):
         for key in topics.get(topic).keySet().toArray():
             t[topic][key] = topics.get(topic).get(key)
     return t
+
+class SimpleNNSearcher:
```

Comment:
`SimpleNearestNeighborSearcher` to be consistent with Java?

Reply:
ok, makes sense